package.json

```diff
@@ -27,6 +27,7 @@
     "codemirror": "^5.54.0",
     "core-js": "^3.6.5",
     "country-iso-3-to-2": "^1.1.0",
+    "cross-env": "^7.0.3",
```

Comment:
Please move this to the `devDependencies` section

Reply:
Yes, of course it should should be in dev. I have made the change and moved it to `devDependencies`.